src/lib.rs

```diff
@@ -34,6 +35,35 @@ lazy_static! {
 pub fn cla_add(cla: Box<dyn ConvergenceLayerAgent>) {
     (*DTNCORE.lock()).cl_list.push(cla);
 }
+pub fn service_add(tag: u8, service: String) {
+    (*DTNCORE.lock()).service_list.insert(tag, service);
+}
+pub fn add_discovery_destination(destination: &String) {
+    (*CONFIG.lock())
+        .discovery_destinations
+        .insert(destination.clone(), 0);
+}
+pub fn update_sequence(destination: &String) {
```

Comment:
whats a sequence in this context? is this the correct term? seems very generic and not ipnd specific .. maybe add docstring or rename a bit

Reply:
if these are for the beacon sequence number, why isn't this logic handled as part of the discovery_destination? sounds like this bsn stuff should be just methods directly on the destination